packages/inertia-vue3/src/link.js

```diff
@@ -75,7 +75,7 @@ export default {
             })
           }
         },
-      }, slots.default())
+      }, slots && slots.default() || '')
```

Comment:
I just ran into the same issue.
From what I have tested you can also just pass `slots` here too.

Reply:
I think that might actually be the better solution
```suggestion
      }, slots)
```